nodes/stampzilla-server/handlers/securewebsockethandler.go

```diff
@@ -252,7 +252,7 @@ func (wsh *secureWebsocketHandler) MessageFromUser(s interfaces.MelodySession, m
 		}).Debug("Received new node configuration")
 
 		wsh.Store.AddOrUpdateNode(node)
-		err = wsh.Store.SaveNodes()
+		err = wsh.Store.SaveNode(wsh.Store.GetNode(node.UUID))
```

Comment:
I did not use `node` but used GetNode since we dont know that all fields are availabe in node since it json value from websocket.